node-notes-cli/app.js

```diff
@@ -0,0 +1,18 @@
+const read = require('./read');
+const add = require('./add');
+const update = require('./update');
+const remove = require('./delete');
+
```

Comment:
You could have required in the data here then passed the data to each of your functions as an argument. Then you wouldn't have to require it in every file. You could even do the same thing with the `fs` module